Simple Lint Problem

### What happened in your environment?

Lint is broken
```
  Running [/home/runner/golangci-lint-2.12.2-linux-amd64/golangci-lint config path] in [/home/runner/work/oras/oras] ...
  Running [/home/runner/golangci-lint-2.12.2-linux-amd64/golangci-lint config verify] in [/home/runner/work/oras/oras] ...
  Running [/home/runner/golangci-lint-2.12.2-linux-amd64/golangci-lint run] in [/home/runner/work/oras/oras] ...
  Error: cmd/oras/root/discover_test.go:35:42: unused-parameter: parameter 'ctx' seems to be unused, consider removing or renaming it as _ (revive)
  func (t *cyclicReferrerTarget) Referrers(ctx context.Context, desc ocispec.Descriptor, artifactType string, fn func(referrers []ocispec.Descriptor) error) error {
                                           ^
  Error: cmd/oras/root/discover_test.go:60:49: unused-parameter: parameter 'referrer' seems to be unused, consider removing or renaming it as _ (revive)
  func (h *recordingDiscoverHandler) OnDiscovered(referrer, subject ocispec.Descriptor) error {
                                                  ^
  Error: internal/graph/graph_test.go:256:32: unused-parameter: parameter 'ctx' seems to be unused, consider removing or renaming it as _ (revive)
  		opts.FindPredecessors = func(ctx context.Context, src content.ReadOnlyGraphStorage, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
  		                             ^
  3 issues:
  * revive: 3
  
  level=warning msg="The linter 'gomodguard' is deprecated (since v2.12.0) due to: new major version. Replaced by gomodguard_v2."
  level=warning msg="Suggested new configuration:\nlinters:\n  enable:\n    - gomodguard_v2\n"
  level=warning msg="[runner/exclusion_rules] Skipped 0 issues by rules: [Text: \"var-naming: avoid package names\", Path: \"internal/\", Linters: \"revive\"]"
  
  Error: issues found
  Ran golangci-lint in 12617ms
```

### What did you expect to happen?

_No response_

### How can we reproduce it?

`make lint`

### What is the version of your ORAS CLI?

main latest

### What is your OS environment?

Any

### Are you willing to submit PRs to fix it?

- [ ] Yes, I am willing to fix it.

--- a/cmd/oras/root/discover_test.go
+++ b/cmd/oras/root/discover_test.go
@@ -32,7 +32,7 @@ type cyclicReferrerTarget struct {
 	referrers map[digest.Digest][]ocispec.Descriptor
 }
 
-func (t *cyclicReferrerTarget) Referrers(ctx context.Context, desc ocispec.Descriptor, artifactType string, fn func(referrers []ocispec.Descriptor) error) error {
+func (t *cyclicReferrerTarget) Referrers(_ context.Context, desc ocispec.Descriptor, _ string, fn func(referrers []ocispec.Descriptor) error) error {
 	return fn(t.referrers[desc.Digest])
 }
 
@@ -57,7 +57,7 @@ type recordingDiscoverHandler struct {
 	count int
 }
 
-func (h *recordingDiscoverHandler) OnDiscovered(referrer, subject ocispec.Descriptor) error {
+func (h *recordingDiscoverHandler) OnDiscovered(_, _ ocispec.Descriptor) error {
 	h.count++
 	return nil
 }

--- a/internal/graph/graph_test.go
+++ b/internal/graph/graph_test.go
@@ -253,7 +253,7 @@ func TestRecursiveFindReferrers(t *testing.T) {
 		descA := ocispec.Descriptor{MediaType: ocispec.MediaTypeImageManifest, Digest: digest.FromString("A"), Size: 1}
 		descB := ocispec.Descriptor{MediaType: ocispec.MediaTypeImageManifest, Digest: digest.FromString("B"), Size: 1}
 		opts := oras.DefaultExtendedCopyGraphOptions
-		opts.FindPredecessors = func(ctx context.Context, src content.ReadOnlyGraphStorage, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
+		opts.FindPredecessors = func(_ context.Context, _ content.ReadOnlyGraphStorage, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
 			switch desc.Digest {
 			case descA.Digest:
 				return []ocispec.Descriptor{descB}, nil

